### PR TITLE
cleanup: inControlBlocked to property

### PR DESCRIPTION
**Pull request by RobozinhoD**
## Purpose / Description
Part of Kotlin cleanup #10489

## Fixes
Part of Kotlin cleanup #10489

## Approach
_How does this change address the problem?_

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration (SDK version(s), emulator or physical, etc)

## Learning (optional, can help others)
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Checklist
_Please, go through these checks before submitting the PR._

- [X] You have a descriptive commit message with a short title (first line, max 50 chars).
- [X] You have commented your code, particularly in hard-to-understand areas
- [X] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/androidTest/java/com/ichi2/anki/testutil/MockReviewerUi.kt [hunk 1]
```diff
@@ -24,9 +24,7 @@ class MockReviewerUi : ReviewerUi {
     override val controlBlocked: ControlBlock?
         get() = null
 
-    override fun isControlBlocked(): Boolean {
-        return false
-    }
+    override val isControlBlocked: Boolean = false
```
Comment (david-allison) [inline AnkiDroid/src/androidTest/java/com/ichi2/anki/testutil/MockReviewerUi.kt:27]: You want to use a property with a custom getter here. search the below for `get()` to see the syntax

https://kotlinlang.org/docs/properties.html

Unit tests are failing due to this

**Review by david-allison — CHANGES_REQUESTED** [review]
Thanks for the contribution! 

You need to provide a custom getter on the property for this to be a full fix

File: AnkiDroid/src/androidTest/java/com/ichi2/anki/testutil/MockReviewerUi.kt (hunk 1 above)
Comment (BrayanDSO) [reply AnkiDroid/src/androidTest/java/com/ichi2/anki/testutil/MockReviewerUi.kt]: Do I need to do it only on this class?

Comment (BrayanDSO) [reply AnkiDroid/src/androidTest/java/com/ichi2/anki/testutil/MockReviewerUi.kt]: Be free to ask anything if you have any questions.

File: AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.kt [hunk 2]
```diff
@@ -2562,9 +2562,7 @@ abstract class AbstractFlashcardViewer :
     override val isDisplayingAnswer
         get() = displayAnswer
 
-    override fun isControlBlocked(): Boolean {
-        return controlBlocked !== ControlBlock.UNBLOCKED
-    }
+    override val isControlBlocked: Boolean = controlBlocked !== ControlBlock.UNBLOCKED
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.kt:2565]: this is incorrect still, as are the other usages.

**Review by david-allison — DISMISSED** [review]

**Comment by RobozinhoD** [comment]
Checks pass now

**Review by BrayanDSO — APPROVED** [review]
LGTM. Thanks for the contribution

**Pull request merged**